plugins/blip/basics/logic/actions.js

```diff
@@ -67,15 +67,21 @@ basicsActions.setSiteChangeEvent = function(sectionName, selectedKey, selectedLa
 
   metricsFunc('Selected ' + selectedLabel);
 
+  var newProfile = _.cloneDeep(this.app.props.patient.profile);
+  if (newProfile.patient.hasOwnProperty('settings')) {
```

Comment:
Ah, I tried that originally, but it didn't _seem_ to work. Looking back, I had the node level wrong. `patient.profile.patient.settings` - OY! 🤕